lib/git_fame/base.rb

```diff
@@ -261,12 +264,19 @@ def convert_exclude_paths_to_array
     end
 
     #
+    # Converts @extensions argument to an array
+    #
+    def convert_extensions_to_array
+      @extensions.split(",")
+    end
```

Comment:
Does this need its own method? I decided to follow the example set by `convert_exclude_paths_to_array`, but not sure if that's the way to go here.